x-pack/scripts/functional_tests.js

```diff
@@ -12,6 +12,7 @@ require('@kbn/test').runTestsCli([
   // require.resolve('../test/reporting/configs/phantom_functional.js'),
   // require.resolve('../test/functional/config.js'),
   // require.resolve('../test/api_integration/config.js'),
+  require.resolve('../test/plugin_api_integration/config.js'),
```

Comment:
This part should be removed before merge: https://github.com/elastic/kibana/pull/23735

But for now it's our best way to do integration tests